app.py

```diff
@@ -102,6 +108,17 @@ def context_change_allowed(self):
         """
         return True
 
+    def create_publish_manager(self, publish_logger=None):
+        """
+        Create and return a :class:`tk_multi_publish2.PublishManager` instance.
+        See the :class:`tk_multi_publish2.PublishManager` docs for details on
+        how it can be used to automate your publishing workflows.
+
+        :param publish_logger: The logger to be used during publish processing.
+        :returns: A :class:`tk_multi_publish2.PublishManager` instance
+        """
+        return self._manager_class(publish_logger=publish_logger)
```

Comment:
not sure this is the right idiom for logging - i think rather than this, we want a fixed logger to always do the logging and then we want to be able to attach a handler we create to that logger. What happens to logging to disk if you bring your own logger for example? 

I think the right way to do this is to be able to get the "address" of the publish logger so that you can then attach a handler to that logger. But we don't want users to provide their own logger.

Reply:
Makes sense. I've gone ahead and removed this for now. I think things will clarify once we plug this into, and rework the UI since that will be a client of this API. If we match the existing behavior with respect to logging, I think it'll be clear what the right interface is.